frontend/src/pages/themes/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { GetStaticProps, NextPage } from 'next';
+import { Themes } from '../../modules/theme/components/Themes';
+
+const ThemePage: NextPage = () => <Themes />;
+
+export const getStaticProps: GetStaticProps = () => {
+  return {
+    props: {
+      dontShowUser: false,
```

Comment:
no need to change but i usually only use this when i want to set it to true. if it's false, there's no need to provide it. but again, no need to change as this is still correct